src/cljs/commiteth/bounties.cljs

```diff
@@ -269,9 +269,9 @@
         [:div.view-loading-container
          [:div.ui.active.inverted.dimmer
           [:div.ui.text.loader.view-loading-label "Loading"]]]
-        [:div.ui.container.open-bounties-container
+        [:div.ui.container.open-bounties-container.
```

Comment:
typo

Reply:
nice catch, thanks! and yes tachyons worked out really well here 🚢